### PR TITLE
Update session-regenerate-id.xml

### PR DESCRIPTION
**Pull request by guy-teube**
improve fr translation

### CODE REVIEW

File: reference/session/functions/session-regenerate-id.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
        <para>
         Si l'on doit effacer l'ancien fichier de session associé ou pas.
         Vous ne devez pas supprimer l'ancienne session si vous avez besoin 
-        d'éviter les courses causées par la suppression ou de détecter/éviter 
+        d'éviter les accès concurrents causés par la suppression, ou de détecter/éviter
```
Comment (devnexen) [inline reference/session/functions/session-regenerate-id.xml:59]: your changes make sense, not sure the comma is necessary tough.

Comment (guy-teube) [reply reference/session/functions/session-regenerate-id.xml:59]: You're right, I've added the comma without really paying attention, I mainly wanted to correct the meaning of the other words. But I don't know how to remove it in the PR.

**Comment by devnexen** [comment]
Thanks for your effort !

File: reference/session/functions/session-regenerate-id.xml (hunk 1 above)
Comment (devnexen) [reply reference/session/functions/session-regenerate-id.xml]: next time, you can add up a new commit and even rebase after into one unique commit.

**Pull request merged**